src/commands/test/test_environment_exceptions.py

```diff
@@ -48,9 +84,30 @@ class StarknetRevertableException(RevertableException):
     The exception is an abstraction over errors raised by StarkNet.
     """
 
+    @staticmethod
+    def extract_error_messages_from_stark_ex_message(
+        msg: Optional[str],
+    ) -> List[str]:
+        results: List[str] = []
+
+        if msg is None:
+            return []
+
+        prefix = "Error message: "
+        remaining_msg = msg
+        start_index = remaining_msg.find(prefix)
+        while not start_index == -1:
+            end_index = remaining_msg.find("\n", start_index)
+            results.append(remaining_msg[start_index + len(prefix) : end_index])
+            remaining_msg = remaining_msg[end_index:]
+            start_index = remaining_msg.find(prefix)
+
+        results.reverse()
+        return results
```

Comment:
We could just use regex
```suggestion
         if msg is None:
            return []
        return [c[0] for c in re.findall("Error Message:((.|\n)+?)((?=Error Message:)|$)", msg)]
```

Reply:
This regex doesn't work in the same way as the code above. I'd rather avoid maintaining regular expressions.

Reply:
I have to re-learn regex every time I use it. (Don't tell anyone.)